listener/Dockerfile

```diff
@@ -110,20 +116,105 @@ RUN mkdir -p build \
   && cmake .. -DUNTRUSTED_ONLY=1 \
   && make
 
+
+# -------------=== common/python build ===-------------
+
+#Build common/cpp intermediate docker image
+FROM python_image as common_python_image
+
+RUN apt-get update \
+ && apt-get install -y -q \
+    make
```

Comment:
Same as https://github.com/hyperledger/avalon/pull/329/files#r397059429

Reply:
This can be addressed in seperate PR.